net/shadowsocks-libev/README.md

```diff
@@ -154,6 +154,8 @@ As for other options, change them only when you know the effect.
 		option src_default 'checkdst'
 		option dst_default 'forward'
 		option local_default 'forward'
+		# option tcp_extra 'tcp dport { 80, 443 }' # tcp only forward connections with dport 80 or 443
+		# option udp_extra 'udp dport { 53 }' # udp only forward connections with dport 53
```

Comment:
Maybe name it with `nft_` prefix , e.g. `nft_tcp_extra` to indicate that they are nft-specific.

Please also update README.md mentioning the additions like we used to have for `ipt_args`.

Reply:
> Maybe name it with `nft_` prefix , e.g. `nft_tcp_extra` to indicate that they are nft-specific.

Done.
 
> Please also update README.md mentioning the additions like we used to have for `ipt_args`.

I don't quite understand, searched the history of `README.md` with `git log -p -- README.md`, and did not find the document of `ipt_args`.

I think the added notes about `nft_tcp_extra/nft_udp_extra` should explain the usage well.

Reply:
My bad, checked again and found that I only added description in `incompatible changes` section for `ipt_args` after the [transition](https://github.com/openwrt/packages/commit/fdaf2de2aee717dc6ae9a39ef9ba6f0388993e5d)...

Anyway, please add proper description for the two options in README.md.  It's supposed to help users understand how components work together and how changing uci options will actually determine behaviors

Reply:
> Anyway, please add proper description for the two options in README.md. It's supposed to help users understand how components work together and how changing uci options will actually determine behaviors

Done, please check the new changes.